b_cfn_sagemaker_endpoint/refresh/source/index.py

```diff
@@ -0,0 +1,68 @@
+import json
+import os
+import time
+from typing import Any, Dict
+import boto3
+
+
+class Counter:
+    def __init__(self, start: int = 0, step: int = 1):
+        self.value = start
+        self.__step = step
+
+    def next(self) -> int:
+        self.value += self.__step
+        return self.value
+
+
+invocation_counter = Counter()
+
+
+def handler(event: Dict[str, Any], context: Any) -> None:
+    invocation_number = invocation_counter.next()
```

Comment:
Overengineered. Limit invocations to 1 and youre good to go.

Reply:
Wasn't aware that such functionality exists. Now will be using `reserved_concurrent_executions=1` to do just that.

Reply:
`reserved_concurrent_executions=1` is doing mirracles, thanks for the tip!